v1/examples/life-events/death-registration-updated.json

```diff
@@ -2,6 +2,8 @@
   "iss": "https://life-events-platform.gov.uk.example",
   "iat": 1691637001,
   "jti": "4d3559ec67504aaba65d40b0363faad8",
+  "sub": "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6",
+  "nbf": 1541493724,
```

Comment:
May be worth making this 1691637001. As the issued at time (IAT) is `Thu Aug 10 2023 04:10:01 GMT+0100` and not before (NBF) is `Tue Nov 06 2018 08:42:04 GMT+0000` and to me it seems a little odd having the NBF so far behind the IAT and I am wondering how realistic this is.